bot/__main__.py

```diff
@@ -1,10 +1,26 @@
+import logging
+
+import aiohttp
+
 import bot
 from bot import constants
-from bot.bot import Bot
+from bot.bot import Bot, StartupError
 from bot.log import setup_sentry
 
 setup_sentry()
 
-bot.instance = Bot.create()
-bot.instance.load_extensions()
-bot.instance.run(constants.Bot.token)
+try:
+    bot.instance = Bot.create()
+    bot.instance.load_extensions()
+    bot.instance.run(constants.Bot.token)
+except StartupError as e:
+    message = "Unknown Startup Error Occurred."
+    if type(e.exception) in [aiohttp.ClientConnectorError, aiohttp.ServerDisconnectedError]:
```

Comment:
This should be an `isinstance` check ~~and it should be a tuple, not a list :P~~